projects/packages/my-jetpack/_inc/components/product-detail-card/index.jsx

```diff
@@ -156,91 +122,75 @@ const ProductDetailCard = ( { slug, onClick, trackButtonClick, className } ) =>
 		}
 	}, [ onClick, trackButtonClick ] );
 
+	// Todo: Fix in product-icons component.
+	const ProductIcon = getIconBySlug( slug ) ? getIconBySlug( slug ) : () => null;
```

Comment:
What exactly happens here?

Reply:
1) Some Products (actually only one so far) don't have associated an icon (Security, because it's bundle product)
2) getIconBySlug() returns `null` when there is not an icon according to the slug ('security')
3) `const ProductIcon` expects a React component (AKA a JS function) but instead of that it receives `null`.
4) `const ProductIcon = getIconBySlug( slug ) ? getIconBySlug( slug ) : () => null;` ensures to define a `noop` function when there is not an icon.

I added a Todo here. It's something that we'd like to improve in a decisive way. https://github.com/Automattic/jetpack/issues/23640

Reply:
@retrofox we could reduce the code here, not?

```javascript
const ProductIcon = getIconBySlug( slug ) || () => null;
```